src/components/Tooltip/PTooltip.vue

```diff
@@ -1,16 +1,14 @@
 <template>
-  <TooltipProvider :v-bind="delegatedProviderProps">
-    <TooltipRoot :v-bind="delegatedRootProps">
```

Comment:
This is where the warning is coming from. These extra colons are binding the object to an attribute called `v-bind`.

<img width="1167" alt="image" src="https://github.com/PrefectHQ/prefect-design/assets/22418768/819fe908-0a14-414b-859e-ccbe50963849">